### PR TITLE
added route for retrieving a particular volume

### PR DESCRIPTION
**Pull request by a-ajith**
The PR contains the addition of routes for retrieving a particular volume with the existing volume

### CODE REVIEW

File: svc-api/router/router.go [hunk 1]
```diff
@@ -252,10 +252,12 @@ func Router() *iris.Application {
 	storage.Get("/{rid}/Drives", system.GetSystemResource)
 	storage.Get("/{rid}/Drives/{rid2}", system.GetSystemResource)
 	storage.Post("/{rid}/Volumes", system.CreateVolume)
+	storage.Get("/{rid}/Volumes/{rid2}", system.GetSystemResource)
```
Comment (jeevan-kamkar) [inline svc-api/router/router.go]: change this to` /{id2}/Volumes/{rid}`

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

**Review by hemantha-beecherla — APPROVED** [review]
LGTM

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Pull request merged**